### PR TITLE
[ARI] Add João Pereira as CLI approver

### PR DESCRIPTION
**Pull request by a-b**
João Pereira, an experienced developer, has made multiple contributions to this project and is actively involved in code reviews and discussions as part of our team.

### CODE REVIEW

**Comment by Gerg** [comment]
Per https://github.com/cloudfoundry/community/blob/main/toc/ROLES.md#promotion-and-revocation, you need 2 existing CLI approvers to review the application and confirm that João meets the criteria for the role.

Once that criteria is met, we can merge this PR.

**Comment by gururajsh** [comment]
LGTM

**Review by Gerg — APPROVED** [review]
Congrats on your promotion!

**Pull request merged**
